src/openrct2/ride/coaster/meta/CompactInvertedCoaster.h

```diff
@@ -42,6 +42,12 @@ constexpr const RideTypeDescriptor CompactInvertedCoasterRTD =
     SET_FIELD(DefaultPrices, { 20, 20 }),
     SET_FIELD(DefaultMusic, MUSIC_STYLE_TECHNO),
     SET_FIELD(PhotoItem, SHOP_ITEM_PHOTO2),
-    SET_FIELD(BonusValue, 80)
+    SET_FIELD(BonusValue, 80),
+    SET_FIELD(ColourPresets, TRACK_COLOUR_PRESETS(
+        { COLOUR_BRIGHT_YELLOW, COLOUR_BRIGHT_YELLOW, COLOUR_BRIGHT_RED },
+        { COLOUR_ICY_BLUE, COLOUR_ICY_BLUE, COLOUR_DARK_GREEN },
+        { COLOUR_BRIGHT_RED, COLOUR_BRIGHT_RED, COLOUR_BRIGHT_YELLOW },
+        { COLOUR_WHITE, COLOUR_WHITE, COLOUR_DARK_GREEN },
+    )),
```

Comment:
You have insistent formatting of this field. Apply the formatting you used here to the other RTDs as well. (So commas after every } and `)),` on its own line.